### PR TITLE
Replace function calls with FQN references

### PR DESCRIPTION
**Pull request by Ocramius**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: php
+cache: false
+
+sudo: false
+
+php:
+  - 7.1
```
Comment (Ocramius) [inline .travis.yml]: Yes

Comment (Ocramius) [inline .travis.yml]: Ref: https://github.com/Ocramius/FunctionFQNReplacer/pull/1/commits/ccb4b2259c947acf47d0d696cc979a743c529ac9

Comment (theofidry) [inline .travis.yml]: 😭

Comment (Ocramius) [inline .travis.yml]: I really really really don't care :-)

Comment (theofidry) [inline .travis.yml]: I understand, just the cries of agony of someone "stuck" at PHP7.0 :P

File: benchmark/AbsoluteFunctionReferenceBench.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public function benchCallUserFuncWithRelativeReference() : void
+    {
+        call_user_func($this->emptyFunction);
+    }
+
+    public function benchCallUserFuncWithAbsoluteReference() : void
+    {
+        \call_user_func($this->emptyFunction);
```
Comment (malukenho) [inline benchmark/AbsoluteFunctionReferenceBench.php]: This `\` is needed?

File: foo.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Roave\FunctionFQNReplacer;
```
Comment (malukenho) [inline foo.php]: It's missing `strict_types`

File: humbug.json.dist [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        "directories": [
+            "src"
+        ]
+    },
+    "timeout": 15,
+    "logs": {
+        "text": "humbuglog.txt",
+        "json": "humbuglog.json"
+    }
+}
```
Comment (malukenho) [inline humbug.json.dist]: EOL EOF

File: benchmark/AbsoluteFunctionReferenceBench.php (hunk 2 above)
Comment (Ocramius) [inline benchmark/AbsoluteFunctionReferenceBench.php]: Yes, it's the exact thing being tested

**Comment by rybakit** [comment]
What about constants? Have you also seen this library: https://github.com/nilportugues/php-backslasher ?

**Comment by Ocramius** [comment]
@rybakit dang, totally missed that! 

Oh well, it has been an interesting coding kata for the day :D

File: src/Console/Command/ReplaceFunctionCalls.php [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+use PhpParser\PrettyPrinter\Standard;
+use Roave\FunctionFQNReplacer\Iterator\PhpFilesInDirectoryIterator;
+use Roave\FunctionFQNReplacer\Lookup\FunctionExists;
+use Roave\FunctionFQNReplacer\NodeVisitor\ReplaceUnqualifiedFunctionCallsWithQualifiedReferences;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ReplaceFunctionCalls extends Command
```
Comment (Ocramius) [inline src/Console/Command/ReplaceFunctionCalls.php]: This is untested, but YOLO. Given that the https://github.com/nilportugues/php-backslasher is tested and stable, I suggest using that for now.

**Pull request merged**